### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zrangerz**
Adding what this project is about, its importance, where to see it rendered.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,21 @@
 
 [![wercker status](https://app.wercker.com/status/71c5fdd2c379b09f18786c1cf741b243/s/master "wercker status")](https://app.wercker.com/project/bykey/71c5fdd2c379b09f18786c1cf741b243)
 
+## What is This Project About?
+This is how we communicate with clients/potential clients in a pre-sales environment. We render the RAML to HTML and provide Javascript mock API services (static/dynamic, stateless/not consistent mocks) to allow external devs to try out our API in a pre-sales, front end app context. We also documentation to walk clients through the major wealth management flows, as we are expert in this and some of our clients aren't. Our first example models Trizic as the Identity Provider (IdP) and the client using the proper routes to onboard a new consumer: https://docs.google.com/document/d/1NVBhHtF3GRsigI9Kca7oidzsM0JF4O78rDsl6DUu6Ow/edit?usp=sharing
```
Comment (DatChive) [inline README.md]: Some rewording suggestions below
```
Javascript -> JavaScript, 
stateless/not consistent -> stateless/non-consistent?, 
front end app context -> front end app? app environment?,
"We also documentation to walk clients" doesn't make sense,
as we are expert in -> as we are an expert in? 
```

File: README.md [hunk 2]
```diff
@@ -2,6 +2,21 @@
...
+There will be a section with all of the official docs at some point, as we build out the workflows, environments, use cases that we explain for customers.
```
Comment (DatChive) [inline README.md]: ```There will be a section with all of the official docs at some point, as we build out the workflows, environments, use cases that we explain for customers.```
Could be...
```There will be a section with all of the official docs at some point as we build out the workflows and environments, that provide explainable use cases to customers```

File: README.md [hunk 3]
```diff
@@ -2,6 +2,21 @@
...
+
+## Prioirty
```
Comment (DatChive) [inline README.md]: ```Prioirty -> Priority```

File: README.md [hunk 4]
```diff
@@ -2,6 +2,21 @@
...
+Since this is now how we communicate to clients, its prod-level important that we text everything. RAML schemas, examples, post/get, request/response bodies and codes need to match the mocks and the docs exactly. We are using Postman to test and to communicate JSON snippets/requests for customers and potential customers (www.getpostman.com) and they must function properly.
```
Comment (DatChive) [inline README.md]:
```
its prod-level -> it's prod-level,
its prod-level important that we text everything -> consider rephrasing. Is it of prod level importance? What does it mean "that we text everything"?,
(www.getpostman.com) and -> (www.getpostman.com), and (can add comma)
```

File: README.md [hunk 5]
```diff
@@ -2,6 +2,21 @@
...
+
+## Offical API RAML->HTML Docs, Mock, UX Locations
+RAML->HTML Docs: http://consumer-redo.trizicdemo.com:8887
+Mock API: http://consumer-redo.trizicdemo.com:8889
+v2 UX: http://consumer-redo.trizicdemo.com
+v3 UX: http://v3.trizic.com
+
+These end points are 2 small AWS instances running Docker containers behind an ELB that update on PR merge every hour on the 5 min (underlying host consumer-redo-1.trizicdemo.com) and 7 min mark (underlying host consumer-redo-2.trizicdemo.com) so it is conceivable that they can have an async moment.
```
Comment (DatChive) [inline README.md]:
```
so it is conceivable that they can have an async moment. -> so it is conceivable that they can be momentarily out of sync ? 
```

File: README.md [hunk 6]
```diff
@@ -2,6 +2,24 @@
 
 [![wercker status](https://app.wercker.com/status/71c5fdd2c379b09f18786c1cf741b243/s/master "wercker status")](https://app.wercker.com/project/bykey/71c5fdd2c379b09f18786c1cf741b243)
 
+## What is This Project About?
+This is how we communicate with clients/potential clients in a pre-sales environment. We render the RAML to HTML and provide JavaScript mock API services (static/dynamic, stateless/not consistent mocks) to allow external devs to try out our API in a pre-sales, front end app context. We now have documentation to walk clients through the major wealth management flows. We are expert in wealth management and the major process flows required and some of our clients aren't.
```
Comment (DatChive) [inline README.md]: - stateless/not consistent -> stateless/non-consistent or inconsistent?
- front end app context -> front end app? app environment?
- we are expert in -> we are an expert in? 

- We are expert in wealth management and the major process flows required and some of our clients aren't. Could be worded "We are an expert in major wealth management process flows, while some of our clients are not."

File: README.md [hunk 7]
```diff
@@ -2,6 +2,24 @@
 
 [![wercker status](https://app.wercker.com/status/71c5fdd2c379b09f18786c1cf741b243/s/master "wercker status")](https://app.wercker.com/project/bykey/71c5fdd2c379b09f18786c1cf741b243)
 
+## What is This Project About?
+This is how we communicate with clients/potential clients in a pre-sales environment. We render the RAML to HTML and provide JavaScript mock API services (static/dynamic, stateless/not consistent mocks) to allow external devs to try out our API in a pre-sales, front end app context. We now have documentation to walk clients through the major wealth management flows. We are expert in wealth management and the major process flows required and some of our clients aren't. 
+
+Our first example models Trizic as the Identity Provider (IdP) and the client using the proper routes to onboard a new consumer: https://docs.google.com/document/d/1NVBhHtF3GRsigI9Kca7oidzsM0JF4O78rDsl6DUu6Ow/edit?usp=sharing
+
+There will be a google docs folder with all of the official docs soon and it will be posted here. We are building out the workflows, environments, use cases and each will spawn a new document.
```
Comment (DatChive) [inline README.md]: - environments, use cases and each will spawn a new document. Oxford comma + and -> environments, and use cases. Each will spawn a new document.

File: README.md [hunk 8]
```diff
@@ -2,6 +2,24 @@
...
+
+## Priority
+Since this is now how we communicate to clients, its of prod-level importance that we test everything. RAML schemas, examples, post/get, request/response bodies and codes need to match the mocks and the docs exactly. We are using Postman to test and to communicate JSON snippets/requests for customers and potential customers (www.getpostman.com), and they must function properly.
```
Comment (DatChive) [inline README.md]: - its -> ```it's```

File: README.md (hunk 6 above)
Comment (jamesdaniello) [reply README.md]: For that last bullet point how about "We are experts in wealth management..."

**Comment by zrangerz** [comment]
Yes please. And automate, deploy, monitor like prod. Those things too.

—
Steve Mays
www.mrmays.com<http://www.mrmays.com>

On May 21, 2018 at 9:20:26 PM, Brad Ondricek (notifications@github.com<mailto:notifications@github.com>) wrote:

we should prob look into adding tests into wercker that validate that the mock server is working with the various examples

**Pull request merged**